src/Symbols/include/Symbols/SymbolUtils.h

```diff
@@ -0,0 +1,25 @@
+// Copyright (c) 2022 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef SYMBOLS_SYMBOL_UTILS_H_
+#define SYMBOLS_SYMBOL_UTILS_H_
+
+#include <filesystem>
+#include <vector>
+
+#include "GrpcProtos/module.pb.h"
+
+namespace orbit_symbols {
+
+// The file extensions for symbols files are .debug for elf files and .pdb for coff files. Only
+// files with following formats are considered `module.sym_ext`, `module.mod_ext.sym_ext` and
```

Comment:
```suggestion
// files with the following formats are considered: `module.sym_ext`, `module.mod_ext.sym_ext` and
```
Might need to reformat.

Reply:
done